Updated text; approved by Lisa through Slack. (#4504)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/users/index.md

- If you don't specify a value for `limit`, the maximum (200) is used as a default.  If you are using a `q` parameter, the default limit is 10.
- An HTTP 500 status code usually indicates that you have exceeded the request timeout. Retry your request with a smaller limit and [paginate](/docs/reference/core-okta-api/#pagination) the results.

> - The `search` parameter delivers optimal performance. Using the `q` or `filter` parameters may yield no results. They may also affect search performance, in which case you should try reformatting the request to use `search`.

Author: ```suggestion
- The `search` parameter delivers optimal performance. Using the `q` or `filter` parameters affect search performance. They may also yield no results, in which case you should try reformatting the request to use `search`.
```

> - The `search` parameter delivers optimal performance. Using the `q` or `filter` parameters may yield no results, or may cause the search to struggle to perform, in which case you should try reformatting the request to use `search`.

Reviewer: ```suggestion
- The `search` parameter delivers optimal performance. Using the `q` or `filter` parameters may yield no results. It may also affect search performance, in which case you should try reformatting the request to use `search`.
```
